pkg/backup/watch.go

```diff
@@ -53,7 +53,7 @@ func (b *Backuper) ValidateWatchParams(watchInterval, fullInterval, watchBackupN
 	if watchBackupNameTemplate != "" {
 		b.cfg.General.WatchBackupNameTemplate = watchBackupNameTemplate
 	}
-	if b.cfg.General.FullDuration.Seconds() < b.cfg.General.WatchDuration.Seconds()*float64(b.cfg.General.BackupsToKeepRemote) {
+	if b.cfg.General.FullDuration.Seconds() > b.cfg.General.WatchDuration.Seconds()*float64(b.cfg.General.BackupsToKeepRemote) {
```

Comment:
are you sure this will help resolve your "connection close" issue?

Reply:
Sorry, I don't understand what exactly you try to fix
this part of code is not related to `current validation allow backups_to_keep_remote less than 4`

could you provide more examples in comments?

Reply:
Sure.

Current (wrong) version in pseudo-code:
```
If FullDurationSeconds < (BackupsToKeep * WatchInterval) {
    raise validation error
}
```

It prevents user from increasing amount of backups - if we set BackupsToKeep to infinity validation starts failing. 

Example:
Let say we want to make incremental daily backups and have full backup on Sunday and to store two weeks of backups on remote.

```
FullDurationSeconds = 1 week (604800 seconds)
BackupsToKeep = 14
WatchInterval = 1 day (86400 seconds)
```

Validation will fail, since `604800 < (14 * 86400)`, it would only work if `BackupsToKeep` is less than 7, which is not enough to store a full week. With the changes it will accept any value of `BackupsToKeep` larger than 7.

Reply:
ok. got it
could you fix intergration_test.go for https://github.com/AlexAkulov/clickhouse-backup/actions/runs/3541456947/jobs/5949819982#step:7:56324

look like your changes broke some backup API scenarios

Reply:
@clamoris any progress?

Reply:
Tests on my machine are ridiculously slow, something is not right
<img width="393" alt="Screenshot 2022-12-02 at 21 52 45" src="https://user-images.githubusercontent.com/2946397/206178460-3c53b74c-db3c-4210-b56b-9b1f0ba43620.png">